### DISCUSSION TITLE
ui.dropdown's Placeholder is not visible when popup == True

### DISCUSSION THREAD

**Discussion by azim-b** [h2oai/wave#1348] [Q&A]
When the list of choices for a ui.dropdown() is large, and `popup=True` by default (because of the size), the placeholder is not visible, unless you click the dropdown for options.

![image](https://user-images.githubusercontent.com/64787868/163197688-4c306f93-a059-4213-99c2-3871b76a1e9a.png)

Wave version 0.20.0.

**Comment by aalencar** [comment]
I can confirm that it's a bug. Tracking here #1349. Thanks @azim-b!
